core/workspace_svg.ts

```diff
@@ -1327,10 +1327,6 @@ export class WorkspaceSvg
       .flatMap((block) => block.getDescendants(false))
       .filter((block) => block.isInsertionMarker())
       .forEach((block) => block.queueRender());
-
-    renderManagement
-      .finishQueuedRenders()
-      .then(() => void this.markerManager.updateMarkers());
```

Comment:
Should `render` no longer call `renderManagement.finishQueuedRenders()` at all?

(Maybe the queued renderers will be finished automatically, and the method call is not needed if you don't need the resulting promise?)

Reply:
Yes, queued renders will finish automatically, and the method call is specifically to be able to get a promise and run after the renders have finished.